api/src/test/java/org/openmrs/api/ConceptServiceTest.java

```diff
@@ -2245,9 +2245,30 @@ public void getConceptsByAnswer_shouldFindAnswersForConcept() {
 	 * @see ConceptService#getConceptsByClass(ConceptClass)
 	 */
 	@Test
-	public void getConceptsByClass_shouldNotFailDueToNoNameInSearch() {
-		ConceptService cs = Context.getConceptService();
-		cs.getConceptsByClass(cs.getConceptClass(1));
+	public void getConceptsByClass_shouldGetConceptsByClass() {
+		// setup
+		ConceptClass cc = new ConceptClass(3);
+		
+		// replay
+		List<Concept> concepts = conceptService.getConceptsByClass(cc);
+		
+		// verify
+		Assert.assertEquals(4, concepts.size());
```

Comment:
Can you verify that you got the 4 concepts that you expected?

Reply:
+1

Reply:
Nice catch @mks-d, just worked on that!